src/main/groovy/io/saagie/plugin/dataops/models/ExportPipeline.groovy

```diff
@@ -21,17 +21,17 @@ class ExportPipeline implements IExists{
         pipelineDTO.setPipelineFromV1ApiResult(pipelineDetailResult)
     }
 
-    void setPipelineVersionFromV1ApiResult(jobs, releaseNote) {
-        pipelineVersionDTO.setPipelineVersionFromV1ApiResult(jobs, releaseNote)
+    void setPipelineVersionFromV1ApiResult(jobs, String releaseNote, String number = null) {
+        pipelineVersionDTO.setPipelineVersionFromV1ApiResult(jobs, releaseNote, number)
     }
 
     void setPipelineVersionFromApiResult(version){
         pipelineVersionDTO.setPipelineVersionFromApiResult(version)
     }
 
-    void addPipelineVersionDTOtoVersions(jobs, releaseNote) {
+    void addPipelineVersionDTOtoVersions(jobs, releaseNote, number = null) {
```

Comment:
It's not important, but you don't respect naming convention because "to" in lowercased
The method name should be `addPipelineVersionDTOToVersions` that is hard to understand, that's why `addPipelineVersionDtoToVersions` would be better